framework/console/controllers/CacheController.php

```diff
@@ -267,11 +267,13 @@ private function findCaches(array $cachesNames = [])
 
             if ($component instanceof CacheInterface) {
                 $caches[$name] = get_class($component);
-            } elseif (is_array($component) && isset($component['class']) && $this->isCacheClass($component['class'])) {
-                $caches[$name] = $component['class'];
+            } elseif (is_array($component) && isset($component['class'])) {
```

Comment:
That is basically the same as it was before, right?

Reply:
yes, but if `$component['class']` is not cache don't try next case because we know that is exactly not cache

Reply:
`&&` already works like that. If condition on the left fails, condition on the right is never executed.

Reply:
I add this then got error with AssetManager after previos commit

Reply:
if `&& $this->isCacheClass($component['class'])`  in that `if` we got `false` and going to next `if` 
In result can be error on`Yii::$app->get($name)`